azure/Guidelines.md

```diff
@@ -46,12 +46,12 @@ The Microsoft Azure Cloud platform exposes its APIs through the core building bl
 
 ### HTTP
 Azure services must adhere to the HTTP specification, [RFC7231](https://tools.ietf.org/html/rfc7231). This section further refines and constrains how service implementors should apply the constructs defined in the HTTP specification. It is therefore, important that you have a firm understanding of the following concepts:
-- [Uniform Resource Locators (URLs)](URLS)
-- HTTP Methods
-- Request & Response Headers
-- Bodies
 
-### Uniform Resource Locators (URLs)
+- Uniform Resource Locators (URLs)
```

Comment:
Since this is a section TOC, should we make these links to the automatically-generated stubs e.g.

```suggestion
- [Uniform Resource Locators (URLs)](#uniform-resource-locators-urls)
```

Reply:
Fixed.

Reply:
Was it supposed to be fixed in the changes you pushed after this? I don't see links for headers.

Also, could you avoid force pushing if necessary e.g., after a rebase which shouldn't always be necessary? It makes it very hard to review just what changed and one has to re-review the PR again.

Reply:
My bad.  Pushed a new commit (not force).  Hopefully fixed now.

Reply:
Verified. Thanks!